ducktape/cluster/remoteaccount.py

```diff
@@ -90,6 +89,14 @@ def kill_process(self, process_grep_str, clean_shutdown=True, allow_fail=False):
             self.ssh(cmd, allow_fail)
 
     def scp_from_command(self, src, dest, recursive=False):
+        if type(src) == list:
```

Comment:
It might be better to switch this test around: instead of checking if it's a list, check if it's not `isinstance(src, basestring)` and then treat it as an iterable. This should work better with other types you could easily encounter. Ideally it would work with iterators too, which don't support `len()`. I think the `len(src) == 1` case should just work using the same code as the `else`, it'll just include the extra `{}`. Not sure about the empty one, but I'm not sure that should even be allowed since it probably produces a command that generates an error anyway since an empty src will result in too few arguments.